### ISSUE TITLE
Nit: "Constructable" vs. "Constructible"

### ISSUE THREAD

**Issue by tomayac** [WICG/construct-stylesheets#90]
I realize this is a bit of a nit and probably arguable, but construct*i*ble seems to be used [a lot more frequently](https://books.google.com/ngrams/graph?content=constructable%2Cconstructible&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cconstructable%3B%2Cc0%3B.t1%3B%2Cconstructible%3B%2Cc0) than construct*a*ble, and the Oxford dictionary even redirects searches for "constructable" to "constructible" (note the linked and final URL of [this search](https://en.oxforddictionaries.com/definition/constructable)). I suggest spelling the two occurrences (one is the actual spec's name) of "constructable" to now read "constructible". For example, #24 also uses this spelling.

**Comment by developit** [comment]
Indeed - I've been moving things over to "Constructible", since that is the US English spelling and generally specs go that route.

**Comment by rakina** [comment]
Thanks! It came up previously on https://github.com/WICG/construct-stylesheets/pull/71. I think we should make the change, but @tabatkins - WDYT?

**Comment by tabatkins** [comment]
English is very... wiggly on -able vs -ible. The *general* rule (with plenty of exceptions) is that you use -able when the root (possibly with a silent -e tacked back on) is a full word on its own (buildable, foldable, comparable, etc), and -ible when the root isn't a full word (edible, etc.)

But it does appear that "constructible" is one of those exceptions, where English speakers have overall decided to use -ible in contravention of the general rule.  That said, "constructable" is recognized as an alternate spelling. (Unlike most -able/-ible words, which definitely have only one accepted spelling.)

So in conclusion, ¯\\\_(ツ)\_/¯

(I'd care more if the word actually showed up in an API, but this is just a spec name.)

**Comment by tomayac** [comment]
To be honest I only noticed because Google Docs gave the word the "red curly underline you misspelled me" treatment. FWIW, my machine is set to speak en-US.

**Comment by domenic** [comment]
IMO this is a lot of churn and change for no real gain. Especially if the spec is going to merge into CSSOM, it's better to just leave as-is.

**Comment by developit** [comment]
I've reverted the update post and demo to use "Constructable"
